src/CommunityToolkit.Maui.Core/Essentials/FileSaver/FileSaver.shared.cs

```diff
@@ -11,13 +13,23 @@ public static class FileSaver
 	public static IFileSaver Default => defaultImplementation.Value;
 
 	/// <inheritdoc cref="IFileSaver.SaveAsync(string, string, Stream, CancellationToken)"/>
+	[SupportedOSPlatform("Android26.0")]
+	[SupportedOSPlatform("iOS14.0")]
+	[SupportedOSPlatform("MacCatalyst14.0")]
```

Comment:
I'm not aware of any problems with this feature on iOS 13 and lower? Does this feature not work on iOS 13, for example? 

Let's remove those version numbers unless we have a known issue on iOS 13 and lower.

If there is a known issue, please open an Issue and comment here with the link.

```suggestion
	[SupportedOSPlatform("iOS")]
	[SupportedOSPlatform("MacCatalyst")]
```

Reply:
our internal FileSaveImplementation contains this attribute, because we use DialogController, that only available in ios14 and later.